### PR TITLE
prune(jsdom): prune old v2 source

### PR DESCRIPTION
**Pull request by peterblazejewicz**
- remove old (years old) v2 from DT
- minor test update

This is to speed up CI tests for Node ecosystem by prunning version
which is no longer expected to be updated on DT itself (and was
published to NPM)

Thanks!

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

### CODE REVIEW

File: types/jsdom/jsdom-tests.ts [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
 
-const dom = new jsdom.JSDOM();
+const dom = new jsdom.JSDOM() || new JSDOM();
 const domWindow = dom.window; // $ExpectType DOMWindow
-
+dom.virtualConsole; // $ExpectType VirtualConsole
+dom.cookieJar; // $ExpectType CookieJar
 domWindow.document.querySelector("slot"); // $ExpectType HTMLSlotElement | null
 domWindow.AbstractRange.prototype; // $ExpectType AbstractRange
 domWindow.StaticRange.prototype; // $ExpectType StaticRange
```
Comment (ExE-Boss) [inline types/jsdom/jsdom-tests.ts:12]: This should probably be done in a separate PR.

File: types/jsdom/jsdom-tests.ts [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
 import jsdom = require("jsdom");
+// tslint:disable-next-line:multiple-import-in-one-file test named export
+import { JSDOM } from "jsdom";
 
-const dom = new jsdom.JSDOM();
+const dom = new jsdom.JSDOM() || new JSDOM();
 const domWindow = dom.window; // $ExpectType DOMWindow
-
+dom.virtualConsole; // $ExpectType VirtualConsole
+dom.cookieJar; // $ExpectType CookieJar
```
Comment (ExE-Boss) [inline types/jsdom/jsdom-tests.ts:9]: **Nit:** I prefer line breaks between grouped statements to match the existing formatting:
```suggestion

dom.virtualConsole; // $ExpectType VirtualConsole
dom.cookieJar; // $ExpectType CookieJar

```

File: types/jsdom/jsdom-tests.ts (hunk 1 above)
Comment (peterblazejewicz) [reply types/jsdom/jsdom-tests.ts]: this was mix of scout badge and avoiding untested badge fate

**Review by ExE-Boss — APPROVED** [review]

**Comment by peterblazejewicz** [comment]
Ready to merge 💘

**Pull request merged**